docs/deployment-examples/asp.net-core-web-application-deployments/index.md

```diff
@@ -70,7 +70,7 @@ public void ConfigureServices(IServiceCollection services)
 
 ## Configuration {#DeployingASP.NETCoreWebApplications-Configuration}
 
-!include <json-configuration-variables>
+!include <structured-configuration-variables>
```

Comment:
@Silcoish the updated file will also be included here. Does the updated content apply to this file (asp.net-core-web-application-deployments/index.md) as well, or do we need to keep the old JSON configuration content for this page?

Reply:
It might be better to link to the new content than duplicate here (if the update applies to this page as well).

Reply:
Yup, it'll apply here too.